cmd/juju/commands/bootstrap.go

```diff
@@ -910,10 +839,10 @@ to create a new model to deploy %sworkloads.
 		Force: c.Force,
 	}
 
-	initialModel, err = c.initializeFirstModel(
-		isCAASController, bootstrapCfg, store, environ, &bootstrapParams,
-	)
-	if err != nil {
+	// This is required so that we don't have to call build-agent. If we
+	// have a currently model, then we can use the agent version from the
```

Comment:
typo "currently"
Can you explain what's happening here? I don't get the `buildagent` reference and the association with SetCurrentModel. Are you referring to the `--build-agent` bootstrap arg? How's that related to whether the client has a current model set?

Reply:
I've removed the comment, it isn't actually required.